apps/examples/testcase/tc_main.c

```diff
@@ -334,6 +345,13 @@ int tc_main(int argc, char *argv[])
 	}
 #endif
 
+#ifdef CONFIG_LIBCXX_UTC
```

Comment:
same as previous

Reply:
I think it is already under #ifdef CONFIG_TASH .